src/Raven.Server/Documents/Replication/ExternalReplicationState.cs

```diff
@@ -28,7 +37,9 @@ public DynamicJsonValue ToJson()
                 [nameof(NodeTag)] = NodeTag,
                 [nameof(LastSentEtag)] = LastSentEtag,
                 [nameof(SourceChangeVector)] = SourceChangeVector,
-                [nameof(DestinationChangeVector)] = DestinationChangeVector
+                [nameof(DestinationChangeVector)] = DestinationChangeVector,
+                [nameof(SourceDatabaseName)] = SourceDatabaseName,
+                [nameof(SourceDatabaseId)] = SourceDatabaseId
```

Comment:
what will happened if the source database failed over?
e,g, shard $0 is on node [A, B]
first node A was sending data, now node B, will we create a new state for that?

Reply:
changed :-)